pages/index.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: homepage
+title: CBA is an extension for chrome browser automation
+menu: Homepage
+---
+
+<img src="/images/Manvel_440x280_border.jpg" alt="CBA banner" />
+<ol>
+  <li>
+    CBA gives ability to automate chrome browser.
+  </li>
+  <li>
+    Ability to setup step-by-step code injections from Extensions popup
+    interface.
+  </li>
+  <li>
+    Ability to Record interactions in the chrome browser and playing them.
+  </li>
+  <li>
+    Projects saving in browser&#39;s local storage.
+  </li>
+  <li>
+    Step by step code injection.
+  </li>
+  <li>
+    Ability of projects import and export.
+  </li>
+  <li>
+    Ability of using CBA ready functions.
+  </li>
+  <li>
+    Ability to write your own code that will have access to web page&#39;sDOM.
+  </li>
+  <li>
+    And much much more...
+  </li>
+</ol>
+
+<section class="box">
+  <h3>Install</h3>
+  <p>
+    <img src="/images/arrow.png" />
+    Get chromium browser automation from google web store.
+  </p>
+  <a
+    href="https://chrome.google.com/webstore/detail/jmbmjnojfkcohdpkpjmeeijckfbebbon"
+    target="_blank"
+    >CBA in chrome web store</a
+  >
+</section>
+
+<section class="box">
+  <h3>Documentation</h3>
+  <p>
+    <img src="/images/doc_grey50.png" />
```

Comment:
Detail: Let's not use too specific image name, i.e. "grey" and "50". Thinking:
- `/images/icon/document.png`
  - in this case we need to move all icons to specific folder
- `/images/document.png`
  - requiers fewer changes